examples/production-cluster-with-metrics.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: "database.arangodb.com/v1alpha"
+kind: "ArangoDeployment"
+metadata:
+  name: "production-cluster"
+spec:
+  metrics:
+    mode: sidecar
+    enabled: true
+    image: arangodb/arangodb-exporter:0.1.7
+    tls: false
+  annotations:                                                                                                                                                  
+    prometheus.io/scrape: 'true'                                                                                                                               
+    prometheus.io/port: '9101'                                                                                                                                 
+    prometheus.io/scrape_interval: '5s'
+  mode: Cluster
+  image: arangodb/arangodb:3.6.1
```

Comment:
Switch to more recent ArangoDB version (you can also do it in all files)

Reply:
Yes will do!